Tangem/Modules/Main/ActionButtons/BaseActionButton/View/ActionButtonView.swift

```diff
@@ -10,31 +10,32 @@ import SwiftUI
 
 struct ActionButtonView<ViewModel: ActionButtonViewModel>: View {
     @ObservedObject var viewModel: ViewModel
+    @Environment(\.isEnabled) var isEnabled
+
+    private var isDisabled: Bool {
+        !isEnabled || viewModel.isDisabled
+    }
 
     var body: some View {
-        Button(
-            action: viewModel.tap,
-            label: {
-                HStack(spacing: 4) {
-                    leadingItem
-                        .frame(width: 20, height: 20)
-                    Text(viewModel.model.title)
-                        .font(Fonts.Bold.subheadline)
-                        .foregroundStyle(Colors.Text.primary1)
-                }
-            }
-        )
+        HStack(spacing: 4) {
+            leadingItem
+                .frame(width: 20, height: 20)
+            Text(viewModel.model.title)
+                .font(Fonts.Bold.subheadline)
+                .foregroundStyle(isDisabled ? Colors.Text.disabled : Colors.Text.primary1)
```

Comment:
Лучше через общий `style` наш

 ```suggestion
                .style(font: Fonts.Bold.subheadline, color: isDisabled ? Colors.Text.disabled : Colors.Text.primary1)
```